app/server.py

```diff
@@ -24,14 +25,18 @@ class PPCGFlask(Flask):
 
 register_filter(Browserify)
 
+
 @server.before_request
 def before_request():
     g.request_start_time = time()
+    jwt_update.delay().wait()
```

Comment:
Is this going to update the keys every request? If so I think we can at least move it to the `/login` endpoint at least so every time a page loads, we don't update jwks 10 times